source/_docs/wordpress-cron.md

```diff
@@ -45,7 +45,7 @@ You can also schedule your own jobs, execute existing jobs, and manage just abou
 One of the first things you'll want to do is test WP-Cron to make sure everything is working correctly. When you execute the command below, make sure to replace SITE_NAME with your site's name from your Pantheon Dashboard and replace ENV_NAME with the desired environment ("dev", "test", "live", or multidev branch name).
 
 ````nohighlight
-$ terminus wp <site>.<env> "cron test"
+$ terminus wp <site>.<env> cron test
```

Comment:
The `--` is relevant even today, as it allows the command to keep working if the user copies the example and adds an option.